scripts/mockdata/objects.json

```diff
@@ -301,5 +301,143 @@
         }
       }
     }
+  },
+  {
+    "name": "Classroom rotation inspector",
+    "position": [0, 0.014, -5.717],
+    "scale": [1, 1, 1],
+    "rotation": [0, 0, 0],
+    "asset_id": 1,
+    "next_objects": [
+      {
+        "id": 2,
+        "action": {
+          "type": "text",
+          "text_id": 1
+        }
+      }
+    ],
+    "texts": ["", "hello", "from the other side"],
+    "is_interactable": true,
+    "animations_json": {
+      "blackboardData": {
+        "componentType": "rotation-controls",
+        "jsonData": {
+          "position": [0, 0, 5]
+        }
+      }
+    }
+  },
+  {
+    "name": "Classroom keypad",
+    "position": [4.487, 1.435, -4.129],
+    "scale": [0.05, 0.05, 0.05],
+    "rotation": [-90.0, 180.0, 180.0],
+    "asset_id": 3,
+    "next_objects": [
+      {
+        "id": 3,
+        "action": {
+          "type": "class",
+          "interactive": true
+        }
+      },
+      {
+        "id": 3,
+        "action": {
+          "type": "text",
+          "text_id": 2
+        }
+      },
+      {
+        "id": 1,
+        "action": {
+          "type": "text",
+          "text_id": 3
+        }
+      }
+    ],
+    "texts": ["", "hello", "kenobi"],
+    "is_interactable": true,
+    "animations_json": {
+      "blackboardData": {
+        "componentType": "keypad",
+        "jsonData": {
+          "password": "1234",
+          "model": "numpad",
+          "is_last_object": true
+        }
+      }
+    }
+  },
+  {
+    "name": "Classroom keyboard",
```

Comment:
Can you rename the objects to map appropriately? For example, this is the Student bedroom keyboard, not the classroom keyboard.